web/src/message_edit.js

```diff
@@ -471,9 +475,10 @@ function edit_message($row, raw_content) {
         }),
     );
 
-    const edit_obj = {$form, raw_content};
-    currently_editing_messages.set(message.id, edit_obj);
-    message_lists.current.show_edit_message($row, edit_obj);
+    const $message_edit_content = $form.find("textarea.message_edit_content");
+    assert($message_edit_content.length === 1);
+    currently_editing_messages.set(message.id, $message_edit_content);
+    message_lists.current.show_edit_message($row, $form);
 
     $form.on("keydown", handle_message_row_edit_keydown);
```

Comment:
So it might also not be a bad idea to just move these handlers to be globally registered as `body.on("keydown", ".message_edit_form", ...`? 

I think these other cleanups are worth it regardless, but it can be less fragile to not be constantly adding/removing handlers on smaller elements.